app/forms/provider_interface/offer_wizard.rb

```diff
@@ -7,7 +7,7 @@ class OfferWizard
     MAX_FURTHER_CONDITIONS = OfferValidations::MAX_CONDITIONS_COUNT - MakeOffer::STANDARD_CONDITIONS.length
 
     attr_accessor :provider_id, :course_id, :course_option_id, :study_mode,
-                  :standard_conditions, :further_conditions, :current_step, :decision,
```

Comment:
Removed an attr_accessor, meaning old caches will break. Need to ignore `further_conditions` from the saved_state

Reply:
Should we temporarily reduce the expiry time of the Wizard Redis cache to be less than 4 hours and aim for deployment is after 7pm to minimize the likeliness of any issues hapenning?

Reply:
I was planning to add a `.except('further_conditions')` to the last saved state. We won't have to worry about deployment then